packages/docs-site/src/components/presenters/code-highlighter/code-highlighter.scss

```diff
@@ -0,0 +1,71 @@
+@import '@royalnavy/css-framework';
+@import 'theme';
+
+$border-radius: 8px;
+$border-radius-button: 4px;
+
+.code-highlighter {
+  display: flex;
+  flex-direction: column;
+  border: 2px solid #768999;
+  border-radius: $border-radius;
+  background-color: #fff;
+  color: #273b5b;
+  overflow: hidden;
+}
+
+.code-highlighter__head {
+  padding: spacing(4);
+  max-height: 20rem;
+  overflow: scroll;
+}
+
+.code-highlighter__body {
+  position: relative;
+  background-color: #0a141b;
+}
+
+.code-highlighter__source {
+  background-color: inherit;
+  font-size: font-size(s);
+
+  pre {
+    overflow: hidden;
+  }
+}
+
+.code-highlighter__copy {
+  position: absolute;
+  top: spacing(2);
+  right: spacing(2);
+  padding: spacing(2);
+  border: none;
+  border-radius: $border-radius-button;
+  background: none;
+  color: #fff;
+  font-size: font-size(s);
+  z-index: 1;
+
+  svg {
+    width: 12px;
+    margin-right: spacing(2);
+  }
+
+  &:focus {
+    outline: none;
+    box-shadow: 0 0 0 2px #346ab1;
+  }
+}
+
+.code-highlighter .line-numbers-rows {
+  padding: 2.5em 0;
```

Comment:
spacing() ?

Reply:
I'm using ems here because the spacing is relative to base styles that come from the line PrismJS line number plugin. This is an extension of their styles.